### PR TITLE
Enable DepthToSpace operator.

### PR DESCRIPTION
**Pull request by arogowie-intel**

### CODE REVIEW

**Comment by arogowie-intel** [comment]
TESTNOW

File: ngraph_onnx/onnx_importer/ops_bridge/opset_01.py [hunk 1]
```diff
@@ -796,6 +796,35 @@ def Split(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> Tup
...
+    n, c, h, w = 1, 1, 1, 1
+    if len(data.shape) == 4:
+        n, c, h, w, = data.shape
+    elif len(data.shape) == 3:
+        c, h, w = data.shape
+    else:
+        raise ValueError('DepthToSpace node (%s): the provided tensor shape (%s) is not supported',
+                         onnx_node.name, str(data.shape))
+
+    tmp = ng.reshape(data, [n, block_size, block_size, c // (block_size ** 2), h, w])
```
Comment (postrational) [inline ngraph_onnx/onnx_importer/ops_bridge/opset_01.py]: Let's give this a better name then `tmp`

File: ngraph_onnx/onnx_importer/ops_bridge/opset_01.py [hunk 2]
```diff
@@ -796,6 +796,35 @@ def Split(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> Tup
...
+    :param onnx_node: The ONNX node representing this operation.
+    :param ng_inputs: The input tensors.
+    :return: Tensor with shape [N, C/(blocksize * blocksize), H * blocksize, W * blocksize].
+    """
+    data = ng_inputs[0]
+    block_size = onnx_node.get_attribute_value('blocksize')
+    if block_size is None:
+        raise ValueError('DepthToSpace node (%s): missing required attribute \"blocksize\"',
+                         onnx_node.name)
+    n, c, h, w = 1, 1, 1, 1
```
Comment (postrational) [inline ngraph_onnx/onnx_importer/ops_bridge/opset_01.py]: Please add a comment why you're setting these values to 1. "Default values for..."

Comment (arogowie-intel) [reply ngraph_onnx/onnx_importer/ops_bridge/opset_01.py]: Add more comments.

**Pull request merged**